project1/css/index.css

```diff
@@ -3,62 +3,244 @@
    License: none (public domain)
 */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+
+   v2.0 | 20110126
+
+   License: none (public domain)
+
+*/
+
+
+
```

Comment:
The only thing I'd add to make your design look more like the image is some white space around the edges of the page (the design currently goes all the way to the top and sides). You could accomplish that in a couple different ways, but I think the best way would be to add a margin on the top/sides for the "container" class that acts as a wrapper around all of the content.